assets/dev/js/frontend/handlers/container/grid-outline.js

```diff
@@ -0,0 +1,229 @@
+export default class GridOutline extends elementorModules.frontend.handlers.Base {
+	isActive() {
+		return elementorFrontend.isEditMode();
+	}
+
+	getDefaultSettings() {
+		return {
+			selectors: {
+				gridOutline: '.e-grid-outline',
+				directGridOverlay: ':scope > .e-grid-outline',
+				boxedContainer: ':scope > .e-con-inner',
+			},
+			classes: {
+				outline: 'e-grid-outline',
+				outlineItem: 'e-grid-outline-item',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		const selectors = this.getSettings( 'selectors' );
+
+		return {
+			outlineParentContainer: null,
+			gridOutline: this.findElement( selectors.gridOutline ),
+			directChildGridOverlay: this.findElement( selectors.directGridOverlay ),
+		};
+	}
+
+	onInit() {
+		super.onInit();
+
+		this.initLayoutOverlay();
+	}
+
+	bindEvents() {
+		elementorFrontend.elements.$window.on( 'resize', this.onDeviceModeChange.bind( this ) );
+		this.addChildLifeCycleEventListeners();
+	}
+
+	unbindEvents() {
+		this.removeChildLifeCycleEventListeners();
+		elementorFrontend.elements.$window.off( 'resize', this.onDeviceModeChange.bind( this ) );
+	}
+
+	initLayoutOverlay() {
+		this.getContainer();
+
+		if ( ! this.elements.outlineParentContainer ) {
+			return;
+		}
+
+		this.removeExistingOverlay();
+		this.createOverlayContainer();
+		this.createOverlayItems();
+	}
+
+	getContainer() {
+		const { grid_outline: gridOutline } = this.getElementSettings();
+
+		this.elements.outlineParentContainer = null;
+
+		if ( gridOutline && '' !== gridOutline ) {
+			return this.getCorrectContainer();
+		}
```

Comment:
```suggestion
		if ( gridOutline ) {
			return this.getCorrectContainer();
		}
```
I think the second part is redundant?

Reply:
Done